obfuscation_test.go

```diff
@@ -137,7 +137,7 @@ func getSpecPubKeys() ([]*btcec.PublicKey, error) {
 	}
 
 	keys := make([]*btcec.PublicKey, len(specPubKeys))
-	for i, sKey := range specPubKeys {
+	for sKey := range specPubKeys {
```

Comment:
That's quite a semantic change, now `sKey` is no longer the string, but the index of the string in the `keys` slice. Does this even compile? The correct fix is this imho:

```golang
for _, sKey := range specPubKeys {
```

Reply:
@cdecker sorry but var not used.

yes, 2 lines equals, but just only look the first is useless

Reply:
.No, I'm saying they are semantically different, see https://github.com/golang/go/wiki/Range#reference for the full reference/